README.md

```diff
@@ -16,7 +16,11 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+"Semantic" is defined as "the function of words beyond their dictionary definition." So the HTML tag itself (e.g. header, p,) has a specific function within HTML beyond its dictionary definition.
+
+HTML Semantic Tags: <div class = "biography"></div> (you could even shorten this to bio)
+
+HTML Non-Semantic Tags: <div class = "blue"></div>
 ```
```

Comment:
:+1:  Yes the main take away here is that Semantic should refer to the function not the form or style of an html element!